python/ray/serve/batching.py

```diff
@@ -523,3 +597,17 @@ async def batch_wrapper(*args, **kwargs):
     # with the underlying function as the sole argument (i.e., it must be a
     # "decorator factory.").
     return _batch_decorator(_func) if callable(_func) else _batch_decorator
+
+
+def _set_result_if_not_done(future: asyncio.Future, result: Any):
+    """Sets the future's result if the future is not done."""
+
+    if not future.done():
+        future.set_result(result)
+
+
+def _set_exception_if_not_done(future: asyncio.Future, exception: Any):
```

Comment:
Really, really loud us to strive for more eloquent names rather than encoding full sentences.

How about `_try_set_exception`?

Reply:
Same applies to setting result

Reply:
Completely disagree -- the existing name is completely self-describing whereas `try_set_exception` doesn't actually tell me what the function does

Reply:
If the function encoded more complex logic then having a more vague name might be appropriate.

Anyways, it's a small taste thing and not that consequential.